### PR TITLE
fix(tabs): Remove gutter from tabs overflow menu

### PR DESCRIPTION
**Pull request by mannycarrera4**
## Summary

Fixes: https://github.com/Workday/canvas-kit/issues/1370

Remove the `margin` on the menu. I also moved the border down one level mostly because I was seeing some visual inconsistency in chrome

category

### CODE REVIEW

**Comment by mannycarrera4** [comment]
Screen Shot 2021-12-07 at 2 39 25 PM
Without the override of the border changes in this PR, this is what the dropdown looks like.

File: modules/react/tabs/lib/menu/MenuList.tsx [hunk 1]
```diff
@@ -39,9 +39,10 @@ export const MenuList = createComponent('div')({
         borderRadius="m"
         padding="zero"
         marginY="zero"
-        marginX="xxs"
+        marginX="zero"
```
Comment (alanbsmith) [inline modules/react/tabs/lib/menu/MenuList.tsx:42]: This could probably just be`margin="zero"` instead of using `marginX` and `marginY`.

**Comment by NicholasBoll** [comment]
I see a small problem in the screenshots. Since the margin was removed, the `MenuItem` elements sit on top of the border radius of the `Menu` element. Do you see that rendering issue in the corners, @mannycarrera4?

**Comment by NicholasBoll** [comment]
It looks like the preview menu avoids this by adding margin to the top/bottom: 
image

**Comment by NicholasBoll** [comment]
The spec has it too: 
image

**Comment by mannycarrera4** [comment]
ah i also see the 1px margin, thanks for this! fixing!

**Comment by NicholasBoll** [comment]
I'm not sure the 1px margin is intentional from the Figma, I think it is trying to avoid the border.

**Review by NicholasBoll — APPROVED** [review]

**Pull request merged**